### PR TITLE
Refactoring code to new css handles and updating tooling

### PR DESCRIPTION
**Pull request by icazevedo**
#### What problem is this solving?

This PR updates the tooling and migrates the components to the new css handles major (1.x)

#### How to test it?

[Workspace](https://icaroshopbycategory--storecomponents.myvtex.com/)

#### Related to / Depends on

[pixel-manager#34](https://github.com/vtex/pixel-manager/pull/34)

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -12,13 +12,13 @@
...
+    "vtex.device-detector": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.device-detector@0.2.6/public/@types/vtex.device-detector",
     "vtex.list-context": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.list-context@0.1.1/public/@types/vtex.list-context",
-    "vtex.native-types": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.native-types@0.7.1/public/@types/vtex.native-types",
+    "vtex.native-types": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.native-types@0.7.4/public/@types/vtex.native-types",
     "vtex.on-view": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.on-view@1.0.0/public/@types/vtex.on-view",
-    "vtex.pixel-manager": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.pixel-manager@1.4.0/public/@types/vtex.pixel-manager",
-    "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.124.3/public/@types/vtex.render-runtime",
+    "vtex.pixel-manager": "https://icaroshopbycategory--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.pixel-manager@1.6.0+build1608064784/public/@types/vtex.pixel-manager",
```
Comment (icazevedo) [inline react/package.json]: Update this after `pixel-manager` is published

File: react/Image.tsx [hunk 2]
```diff
@@ -81,15 +79,20 @@ function Image(props: ImageProps) {
...
+
   const imageRef = useRef<HTMLImageElement | null>(null)
   const isLoaded = useImageLoad(imageRef, {
     bailOut: !experimentalPreventLayoutShift,
   })
+
   const intl = useIntl()
-  const handles = useCssHandles(CSS_HANDLES, {
+  const { handles } = useCssHandles(CSS_HANDLES, {
     migrationFrom: 'vtex.store-components@3.x',
```
Comment (klzns) [inline react/Image.tsx:92]: Is this working properly?

Comment (icazevedo) [reply react/Image.tsx:92]: Good question, let me check

**Review by vitorflg — APPROVED** [review]

File: react/Image.tsx (hunk 2 above)
Comment (icazevedo) [reply react/Image.tsx:92]: There's an image [here](https://icaroshopbycategory--storecomponents.myvtex.com/about-us) (VTEX logo). I've added custom classes to the image component and the behavior is that it overrides both the default handle and the migrationFrom handle.

Screen Shot 2020-12-15 at 6 57 44 PM

Screen Shot 2020-12-15 at 6 59 37 PM

**Review by victorhmp — APPROVED** [review]

**Pull request merged**